.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: scala
+jdk: oraclejdk8
+scala: 2.10.6
+script: sbt verify
```

Comment:
where does `verify` come from? I'd say use `test` until this has some `scripted` tests